packages/lesswrong/lib/crud/withMulti.ts

```diff
@@ -287,8 +287,7 @@ export function useMulti<
   loading: boolean,
   loadingInitial: boolean,
   loadingMore: boolean,
-  // TODO: Should be nullable
-  results: Array<FragmentTypes[FragmentTypeName]>,
+  results?: Array<FragmentTypes[FragmentTypeName]>,
```

Comment:
No fancy union type here. The presence of loadMores makes it so loading can overlap with results being present.